identity-wallet/src/command.rs

```diff
@@ -90,6 +78,24 @@ pub async fn handle_action<R: tauri::Runtime>(
     Result::Ok(())
 }
 
+pub async fn handle_action<R: tauri::Runtime>(
+    action: Action,
+    app_handle: tauri::AppHandle<R>,
+    container: tauri::State<'_, AppStateContainer>,
+    window: tauri::Window<R>,
+) -> Result<(), String> {
+    tokio::select! {
+        res = main_exec(action, app_handle, container, window) => {
+            info!("Finish invoke");
+            res
+        }
+        _ = deadlock_safety() => {
+            error!("Operation timed out");
```

Comment:
Do you think we should log as a `warn!()` instead?

Reply:
But nothing got updated so it's an error?

Reply:
It doesn't matter right? A warning is something that is not correct but still works like missing env var but now its using a default value for that, an error is the intended action has not been successfully executed